website/docs/cli/commands/vendor/vendor-pull.mdx

```diff
@@ -108,6 +108,112 @@ The following config can be used to download the `vpc` component from an AWS pub
       version: "latest"
   ```
 
+## Vendoring from SSH
+
+This is the most common way to access non-public Git repositories from automated systems because it allows access to private repositories without interactive prompts.
+Atmos will use any installed SSH keys automatically. There are two primary ways to specify an SSH source:
+
+### Implicit SSH
+
+When no explicit scheme is provided, for example:
+
+```shell
+git::git@github.com:cloudposse/terraform-null-label.git?ref={{.Version}}
+```
+
+Atmos’s rewrites this URL to:
+
+```shell
+git::ssh://git@github.com/cloudposse/terraform-null-label.git?depth=1&ref={{.Version}}
+```
+
+In this case, Atmos injects a default username ("git").
+
+### Explicit SSH
+
+When the SSH scheme is specified explicitly, for example:
+
+```shell
+git::ssh://git@github.com/cloudposse/terraform-null-label.git?ref={{ .Version }}
+```
+
+the URL is used as provided. In this scenario, no rewriting occurs because the URL already includes an explicit `ssh://` schema.
+
+### Important Notes
+
+- A URL like:
+  ```shell
+  github.com:cloudposse/terraform-null-label.git?ref={{ .Version }}
+  ```
+  is **not valid** because go-getter misinterprets **github.com:** as a schema and will error out.
+
+- A URL without any scheme, such as:
+  ```shell
+  github.com/cloudposse/terraform-null-label.git?ref={{ .Version }}
+  ```
+  defaults to HTTPS and triggers Atmos to inject credentials if available.  
+
+## HTTPS Vendoring
+
+Atmos supports vendoring components using HTTPS links. 
+
+For example:
+```
+github.com/cloudposse/terraform-null-label.git?ref={{ .Version }}
+```
+is automatically resolved as:
+```
+git::https://github.com/cloudposse/terraform-null-label.git?depth=1&ref={{ .Version }}
+```
+
+## Authentication & Token Usage for HTTPS
+
+Atmos prioritizes authentication credentials based on predefined environment variables. The priority order for each provider is:
+
+### GitHub
+| Environment Variable           | Purpose                                                                                                  |
+|--------------------------------|----------------------------------------------------------------------------------------------------------|
+| ATMOS_GITHUB_TOKEN             | Bearer token for GitHub API requests, enabling authentication for private repositories and higher rate limits. |
+| GITHUB_TOKEN                   | Used as a fallback if ATMOS_GITHUB_TOKEN is not set.                                                    |
```

Comment:
Please use definition lists instead of tables. Older documentation that we haven't updated still uses tables.

```suggestion
<dl>
  <dt>`ATMOS_GITHUB_TOKEN`</dt>
  <dd>
    Bearer token for GitHub API requests, enabling authentication for private repositories and higher rate limits. The `ATMOS_GITHUB_TOKEN` takes precedence over the `GITHUB_TOKEN`.
    
    <dt>Default Username</dt>
    <dd>`x-access-token`</dd>
  </dd>
  <dt>`GITHUB_TOKEN`</dt>
  <dd>Used as a fallback if `ATMOS_GITHUB_TOKEN` is not set.</dd>
</dl>
```